keyboards/ReverseStudio/DecadePad/DecadePad.h

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+#include "quantum.h"
+#define LAYOUT( \
```

Comment:
This is actually a supported layout for the community layouts. 

```suggestion

#define LAYOUT_numpad_6x4( \
```